src/components/AppCard/edit/editCard.tsx

```diff
@@ -1,32 +1,78 @@
-import React, { useState } from "react";
-import binIcon from "../../../images/Kind=Delete.svg";
-import styles from "./editCard.module.css";
+import React, { useState } from 'react';
+import binIcon from '../../../images/Kind=Delete.svg';
+import styles from './editCard.module.css';
```

Comment:
**nitpick:** Component filenames should be written with `PascalCase` not `camelCase`